src/model/mitra-activity.ts

```diff
@@ -1,10 +1,10 @@
 import { ActivityObject, Link } from "activitypub-objects";
-import { Activity } from "activitypub-objects/dst/activities/activity";
+import { Activity as ActivityInterface } from "activitypub-objects/dst/activities/activity";
```

Comment:
Sorry, now that I had a hands on I understand what this is about. So my final suggestion would be:
```suggestion
import { Activity as BaseActivity } from "activitypub-objects/dst/activities/activity";
```